client/src/features/bulletins/host/BulletinEdit/BulletinEdit.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { useState, useEffect, useCallback } from 'react';
```

Comment:
Invece di riduplicare tutto potevi rendere il form di creazione più generico ed accettare una prop con i dati iniziali del form, che possono essere o quelli di default in creazione o quelli del bulletin esistente in modifica. Questo considerando che non prevediamo differenze in futuro tra creazione e modifica, altrimenti avrebbe senso mantenere due versioni separate.